clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -22,6 +22,9 @@ def err_drv_unsupported_option_argument : Error<
   "unsupported argument '%1' to option '%0'">;
 def err_drv_unsupported_option_argument_for_target : Error<
   "unsupported argument '%1' to option '%0' for target '%2'">;
+def err_drv_unsupported_opt_removed : Error<
```

Comment:
There is an existing `err_drv_unsupported_opt` [here](https://github.com/intel/llvm/blob/1e45bb44197997cc46acd5a3303f6840e5f5670f/clang/include/clang/Basic/DiagnosticDriverKinds.td#L14C1-L15C27)
Can that be used instead of a new one?

Reply:
Given the quick turnaround and removal of the options (without going through the typical 'deprecation' state) it was agreed upon through the Options working group to use this new diagnostic to better inform any users of the change instead of just emitting a 'unsupported option' diagnostic.

Reply:
Is it possible to update the definition of `err_drv_unsupported_opt` to add the additional text?
The wording seems generic and I wonder it might not impact other areas of the code where `err_drv_unsupported_opt` is used.

Reply:
We want to only treat the FPGA specific options as listed in the description with this new diagnostic.  Once the dust settles and we have moved all users to the FPGA specific compiler we can move to treating these as regular unsupported options.